com.unity.multiplayer.mlapi/Runtime/Messaging/Buffering/PreBufferPreset.cs

```diff
@@ -8,7 +8,7 @@ internal struct PreBufferPreset
         public byte MessageType;
         public bool AllowBuffer;
         public ulong ClientId;
-        public Channel Channel;
```

Comment:
I thought Channel was going to change to NetworkDelivery?

Reply:
yeah, we have multiple types of channel things:

- (enum) `ChannelType` → `NetworkDelivery`
  - Unreliable
  - UnreliableSequenced
  - etc.
- (enum) `Channel` → `NetworkChannel`
  - Internal
  - TimeSync
  - ReliableRpc
  - UnreliableRpc
  - etc.
- (class) `TransportChannel`
  - `NetworkChannel Id;`
  - `NetworkDelivery Type;`